<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AscentDialog</class>
 <widget class="QDialog" name="AscentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New ascent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Ascent title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Optional</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="peakLabel">
       <property name="text">
        <string>Peak</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="peakLayout">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="2" column="0">
        <widget class="QComboBox" name="peakCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="regionFilterCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Filter peaks by region</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="newPeakButton">
         <property name="text">
          <string>New peak...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/peak.svg</normaloff>:/icons/peak.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="dateSelectionLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="dateCheckbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximumDate">
          <date>
           <year>2100</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>1900</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="dateSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="peakIndexLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QSpinBox" name="peakIndexSpinner">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="suffix">
            <string notr="true">.</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="peakIndexLabel">
           <property name="text">
            <string>peak on that day</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="dateSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Local time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="timeLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="timeCheckbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="timeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="elevationGainLabel">
       <property name="text">
        <string>Elevation gain</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="elevationGainLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QCheckBox" name="elevationGainCheckbox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="elevationGainSpinner">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string notr="true"> m</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>9000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="elevationGainSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hikeKindLabel">
       <property name="text">
        <string>Kind of hike</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="hikeKindLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="hikeKindCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hikeKindSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="traverseCheckbox">
         <property name="text">
          <string>Traverse</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hikeKindSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="difficultyLabel">
       <property name="text">
        <string>Difficulty</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="difficultyLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="difficultySystemCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Select system</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="difficultyGradeCombo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>None</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="tripLabel">
       <property name="text">
        <string>During trip</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="tripSelectionLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="tripCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>None</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newTripButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New trip...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/trip.svg</normaloff>:/icons/trip.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="hikersLabel">
       <property name="text">
        <string>Participants</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="hikersLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QListView" name="hikersListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>76</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>112</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>18</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="hikerButtonLayout">
         <item>
          <widget class="QToolButton" name="addHikerButton">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string notr="true">+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeHikersButton">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="hikersSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="photosLabel">
       <property name="text">
        <string>Photos</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QVBoxLayout" name="photosLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="photosListLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QListView" name="photosListView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>94</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>130</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::InternalMove</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="photosToolLayout">
           <item>
            <widget class="QToolButton" name="addPhotosButton">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removePhotosButton">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="photoDescriptionLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Photo description</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descriptionEditor">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="bottomLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonLayout">
     <property name="spacing">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Add ascent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
